servant-auth-docs/servant-auth-docs.cabal

```diff
@@ -15,7 +15,7 @@ maintainer:     jkarni@gmail.com
 copyright:      (c) Julian K. Arni
 license:        BSD3
 license-file:   LICENSE
-tested-with:    GHC == 7.10.3, GHC == 8.0.2, GHC == 8.2.2, GHC==8.4.4, GHC==8.6.2
```

Comment:
Then you really should tweak `base` lower bound

Reply:
https://matrix.hackage.haskell.org/#/package/servant-auth-server <- you does support them, so you should test or explicitly drop support